docs/development.md

```diff
@@ -170,7 +170,7 @@ It is recommended to use Firefox 46.0 as this is the browser used in CI build (T
 * You need to use chromedriver for testing with Chrome.
   * Download the latest stable chromedriver from [here](https://sites.google.com/a/chromium.org/chromedriver/downloads).
     The site will also inform the versions of Chrome that can be used with the driver.
-  * Specify the path to the chromedriver executable in `test.chromedriver.path` value in `test.properties`.
+  * Specify the path to the chromedriver executable in `test.chromedriver.path` value in `test.template.properties`.
```

Comment:
Please revert this change as it's unrelated to the issue. Also, I believe you now know [why it's wrong](https://github.com/TEAMMATES/teammates/pull/7572#issuecomment-310674892)?